website/content/docs/operations/federation/failure.mdx

```diff
@@ -0,0 +1,138 @@
+---
+layout: docs
+page_title: Federated Cluster Failure Scenarios
+description: Failure scenarios in multi-region federated cluster deployments.
+---
+
+# Failure Scenarios
+
+When running Nomad in federated mode, failure situations and impacts are different depending on
+whether the authoritative region is the impacted region or not, and what the failure mode is. In
+soft failures, the region's servers have lost quorum but the Nomad processes are still up, running,
+and reachable. In hard failures, the regional servers are completely unreachable and are akin to
+the underlying hardware having been terminated (cloud) or powered-off (on-prem).
+
+The scenarios are based on a Nomad deployment which is running with three federated regions named
+`asia-south-1`, `europe-west-1`, and `us-east-1`. The region `europe-west-1` is authoritative.
+
+## Federated Region Failure: Soft
+In this situation the region `asia-south-1` has lost leadership but the servers are reachable and
+up.
+
+All server logs in the impacted region will have entries such as:
```

Comment:
```suggestion
All server logs in the impacted region have entries such as this example.
```

[Introduce code block with an descriptive, imperative sentence that ends with a period](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#code-blocks)